packages/test/snapshots/src/test/serialized.spec.ts

```diff
@@ -0,0 +1,135 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import fs from "fs";
+import {
+    Loader,
+} from "@fluidframework/container-loader";
+import {
+    LocalCodeLoader,
+    TestFluidObjectFactory,
+    TestFluidObject,
+    LoaderContainerTracker,
+} from "@fluidframework/test-utils";
+import { IFluidCodeDetails } from "@fluidframework/core-interfaces";
+import { SharedMap, SharedDirectory } from "@fluidframework/map";
+import { SharedString, SparseMatrix } from "@fluidframework/sequence";
+import { SharedCell } from "@fluidframework/cell";
+import { Ink } from "@fluidframework/ink";
+import { SharedMatrix } from "@fluidframework/matrix";
+import { SharedCounter } from "@fluidframework/counter";
+import { ConsensusRegisterCollection } from "@fluidframework/register-collection";
+import { ConsensusQueue, ConsensusOrderedCollection } from "@fluidframework/ordered-collection";
+import { describeNoCompat } from "@fluidframework/test-version-utils";
+import { ChildLogger } from "@fluidframework/telemetry-utils";
+
+describeNoCompat(`Container Serialization Backwards Compatibility`, (getTestObjectProvider) => {
```

Comment:
fine to reuse this. but these tests could just be hard coded against local server, as the server doesn't really matter